source/sass/views/blog.scss

```diff
@@ -17,3 +17,9 @@
     }
   }
 }
+
+a.blog-detailed-category {
```

Comment:
Do we need "blog-detailed" here? Can call this just `category` without running into conflicts?

Reply:
Looks like that's completely possible—updating now.